docs/app/data/licenses.json

```diff
@@ -2,161 +2,161 @@
for any direct, special, indirect, incidental, consequential, punitive, exemplary, or other losses, costs, expenses, or damages arising out of this Public License or use of the Licensed Material, even if the Licensor has been advised of the possibility of such losses, costs, expenses, or damages. Where a limitation of liability is not allowed in full or in part, this limitation may not apply to You.\r\nThe disclaimer of warranties and limitation of liability provided above shall be interpreted in a manner that, to the extent possible, most closely approximates an absolute disclaimer and waiver of all liability.\r\nSection 6 - Term and Termination.\r\n\r\nThis Public License applies for the term of the Copyright and Similar Rights licensed here. However, if You fail to comply with this Public License, then Your rights under this Public License terminate automatically.\r\nWhere Your right to use the Licensed Material has terminated under Section 6(a), it reinstates:\r\n\r\nautomatically as of the date the violation is cured, provided it is cured within 30 days of Your discovery of the violation; or\r\nupon express reinstatement by the Licensor.\r\nFor the avoidance of doubt, this Section 6(b) does not affect any right the Licensor may have to seek remedies for Your violations of this Public License.\r\nFor the avoidance of doubt, the Licensor may also offer the Licensed Material under separate terms or conditions or stop distributing the Licensed Material at any time; however, doing so will not terminate this Public License.\r\nSections 1, 5, 6, 7, and 8 survive termination of this Public License.\r\nSection 7 - Other Terms and Conditions.\r\n\r\nThe Licensor shall not be bound by any additional or different terms or conditions communicated by You unless expressly agreed.\r\nAny arrangements, understandings, or agreements regarding the Licensed Material not stated herein are separate from and independent of the terms and conditions of this Public License.\r\nSection 8 - Interpretation.\r\n\r\nFor the avoidance of doubt, this Public License does not, and shall not be interpreted to, reduce, limit, restrict, or impose conditions on any use of the Licensed Material that could lawfully be made without permission under this Public License.\r\nTo the extent possible, if any provision of this Public License is deemed unenforceable, it shall be automatically reformed to the minimum extent necessary to make it enforceable. If the provision cannot be reformed, it shall be severed from this Public License without affecting the enforceability of the remaining terms and conditions.\r\nNo term or condition of this Public License will be waived and no failure to comply consented to unless expressly agreed to by the Licensor.\r\nNothing in this Public License constitutes or may be interpreted as a limitation upon, or waiver of, any privileges and immunities that apply to the Licensor or You, including from the legal processes of any jurisdiction or authority.\r\n"
```

Comment:
Were these changes intentional?

Reply:
Hmm, nope, must have been a grunt task perhaps? Anyway, I will revert the changes.